### PR TITLE
Hack utility added

### PR DESCRIPTION
**Pull request by Madhan-SWE**
Hack utility added to
- update gomock
- update good
- verify gofmt
- verify gofmt
- verify vendor
- verify covet

### CODE REVIEW

File: hack/update-gofmt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
```
Comment (mkumatag) [inline hack/update-gofmt:3]: fix the copyright year and comment applies for all the other copyright headers as well

File: hack/verify-govet [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+echo "Verifying govet"
+
+go vet $(go list ./... | grep -v vendor)
+
+echo "Done"
```
Comment (mkumatag) [inline hack/verify-govet:23]: add a newline

File: pkg/driver/controller_test.go [hunk 3]
```diff
@@ -21,11 +21,11 @@ import (
...
 	csi "github.com/container-storage-interface/spec/lib/go/csi"
 	"github.com/golang/mock/gomock"
-	"github.com/ppc64le-cloud/powervs-csi-driver/pkg/cloud"
-	mocks "github.com/ppc64le-cloud/powervs-csi-driver/pkg/cloud/mocks"
-	"github.com/ppc64le-cloud/powervs-csi-driver/pkg/util"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	"sigs.k8s.io/ibm-powervs-block-csi-driver/pkg/cloud"
+	mocks "sigs.k8s.io/ibm-powervs-block-csi-driver/pkg/cloud/mocks"
+	"sigs.k8s.io/ibm-powervs-block-csi-driver/pkg/util"
```
Comment (mkumatag) [inline pkg/driver/controller_test.go:28]: these imports are fixed in https://github.com/ppc64le-cloud/ibm-powervs-block-csi-driver/pull/85

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: hack/update-gofmt (hunk 1 above)
Comment (Madhan-SWE) [reply hack/update-gofmt:3]: @mkumatag Should I make it as 2022?

Comment (Madhan-SWE) [reply hack/update-gofmt:3]: The copyright headers should contain the year at which it is created right?

File: pkg/driver/controller_test.go (hunk 3 above)
Comment (Madhan-SWE) [reply pkg/driver/controller_test.go]: rebased code. Comment outdated.

File: hack/verify-govet (hunk 2 above)
Comment (Madhan-SWE) [reply hack/verify-govet]: fixed

File: hack/update-gofmt (hunk 1 above)
Comment (Madhan-SWE) [reply hack/update-gofmt]: I changed the year to 2022

Comment (mkumatag) [reply hack/update-gofmt]: yes, 2022

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**